sds_data_manager/stacks/step_function_stack.py

```diff
@@ -25,13 +24,10 @@ class ProcessingStepFunctionStack(Stack):
     def __init__(
         self,
         scope: Construct,
-        id: str,
```

Comment:
These removed parameters need a removed docstring too, and construct_id needs a new one